starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecUtil.java

```diff
@@ -659,35 +656,20 @@ public static String findVideoDecoder(
       // Enable the improved support check based on more specific APIs, like isSizeSupported() or
       // areSizeAndRateSupported(), for 8k content. These APIs are theoretically more accurate,
```

Comment:
You can remove this comment now since the boolean is gone.

Reply:
Done.